connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -1658,6 +1659,24 @@ class SparkConnectPlanner(val session: SparkSession) {
         Some(
           CatalystDataToProtobuf(children.head, messageClassName, binaryFileDescSetOpt, options))
 
+      case "percentile_cont" if Seq(2, 3).contains(fun.getArgumentsCount) =>
```

Comment:
do we plan to support `percentile_cont` and `percentile_disc`? they are not [in the list](https://issues.apache.org/jira/browse/SPARK-43907)

if we support them, shall we add them to `FunctionRegistry ` ?

@HyukjinKwon @cloud-fan